helm/app-admission-controller/values.yaml

```
@@ -1,11 +1,12 @@
+provider:
+  kind: ""
```

I think this value should not be an empty string. Can you change this to `aws`?

---

My personal preference would be to fail on start instead of having wrongly configured provider by default.

---

Yes agree better for it to fail. Otherwise we'll change the config and end up using aws for the wrong provider.